parm/met_config/MODEConfig_wrapped

```diff
@@ -237,7 +237,7 @@ nc_pairs_flag   = {
    object_raw   = TRUE;
    object_id    = TRUE;
    cluster_id   = TRUE;
-   polylines    = TRUE;
+   polylines    = FALSE;
```

Comment:
Why was this changed? This change will likely break all of the existing use cases that run MODE because the output will differ. If you need to override this value for a specific use case, you can set this value in the .conf file:

MODE_MET_CONFIG_OVERRIDES = nc_pairs_flag = {polylines    = FALSE;}

Reply:
This was changed to make the files CF compliant so they can be easily read back into MET.   Let me try doing this, as I've never done this.  Do I need to make edits to the config file for this to work?

Reply:
Actually it looks like your use case doesn't even use this file, so you should be able to just change this value back to the default.

Reply:
The use case does use the output netcdf file

Reply:
The use case uses this file:

MODE_CONFIG_FILE = {CONFIG_DIR}/MODEConfig_BrightnessTemp

not parm/met_config/MODEConfig_wrapped

Reply:
Hmmm, I thought I only changed this in the case use file, not MODEConfig_wrapped.  LEt me check on this